doc/reference/cmdline.rst

```diff
@@ -15,7 +15,8 @@ Running the server
 
 .. option:: -d <database>, --database <database>
 
-    database used when installing or updating modules.
+    database(s) used when installing or updating modules.
+    Providing a coma-separated list restrict access to databases provided in list.
```

Comment:
com*m*a. and restrict*s*.

Also should have a newline between this line and the previous one to have two different paragraphs, otherwise they'll be parsed as a single paragraph.